recipes/jcc/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export JCC_JDK=$JAVA_HOME
+
+if [ "$(uname)" == "Darwin" ]
+then
+    # OSX recipe
+    # export LD_LIBRARY_PATH=$PREFIX/jre/lib/amd64/server:$PREFIX/jre/lib/amd64:$LD_LIBRARY_PATH
+    export MACOSX_DEPLOYMENT_TARGET=10.9
+    export MACOSX_VERSION_MIN=10.9
```

Comment:
Hi, many thanks for the comments!
I had some issues with building on travis that defaulted to 10.5 so I added them here... Will test if the issue is still there.